server.go

```diff
@@ -552,10 +552,10 @@ func (p sshFxpFsetstatPacket) respond(svr *Server) error {
 		}
 	}
 
-	return svr.sendPacket(statusFromError(p.Id, err))
+	return svr.sendPacket(statusFromError(p.ID, err))
 }
 
-func errnoToSshErr(errno syscall.Errno) uint32 {
+func errnoToSSHErr(errno syscall.Errno) uint32 {
```

Comment:
I don't think renaming this has improved the name of this function, and it wasn't good to start with. If you want to come up with a new name, that would be great.